.github/workflows/awsfulltest.yml

```diff
@@ -9,6 +9,16 @@ on:
     types: [completed]
   workflow_dispatch:
 
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
On my phone now so tricky to see, but I think I remembered seeing that we already had these defined lower in these files? So that needs to be deleted as we don't need it twice..